docs/reference/search-api.md

```diff
@@ -107,13 +102,16 @@ The endpoint will return 10 million values if 10 million documents match the que
 |----------|------|-------------|---------------|
 | **query** | `String` | Query text. See the [query language doc](query-language.md) (mandatory) | |
 | **fastField** | `String` | Name of a field to retrieve from documents. This field must be marked as "fast" in the index config. (mandatory)| |
-| **searchFields** | `[String]` | If set, specify the set of fields the search will be performed on | |
+| **searchFields** | `[String]` | Fields to search on. Comma-separated list, e.g. "field1,field2" | index_config.search_settings.default_search_fields    |
 | **startTimestamp** | `i64` | If set, restrict search to documents with a `timestamp >= start_timestamp` | |
 | **endTimestamp** | `i64` | If set, restrict search to documents with a `timestamp < end_timestamp`` | |
 | **outputFormat** | `String` | Response output format. `csv` or `clickHouseRowBinary`  | `csv` |
-| **tags** | `[String]` | If set, the search is restricted to only splits having one of the tags | |
 
 
 ### Response
 
-The response is a list of all the field values from documents matching the query. The field must be marked as "fast" in the index config for this to work. The formatting is based on the specified output format. 
+The response is a list of all the field values from documents matching the query. The field must be marked as "fast" in the index config for this to work. 
+The formatting is based on the specified output format. 
+
+On error a "X-Stream-Error" header will be sent via the the trailers channel with information about the error and the stream will be closed. Depending on the client, the trailer header may not be shown. 
```

Comment:
This is hard to understand. We need to be more accurate and say that we are using a`Streaming HTTP protocol`.

It's important to talk about the HTTP trailer and also about the `send.abort()`.

Moreover I think HTTP trailers are only compatible with HTTP2.

If a client cannot see the trailer, what will happen? You can maybe add a note on the logging error.

You have to assume that a user has little knowledge about this and if we are not accurate, he will think there is a bug and quit.

Reply:
As I understand, we are using the chunked transfer encoding, which is not supported by HTTP2.
So I don't understand how it can work with the HTTP trailers which are only HTTP2

Reply:
Maybe we are doing something wrong :/
Can you check that ?

Reply:
Trailers are HTTP1.1 https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Trailer.
Not sure how they behave if the client switches to HTTP2 though